_data/utilities.yml

```diff
@@ -73,9 +73,8 @@ websites:
       tfa: No
 
     - name: Dominion
```

Comment:
It appears they've [rebranded](https://www.dominionenergy.com/NewBrand) as "Dominion Energy", so I think we should update their name to match that.